rero_ils/modules/items/api/record.py

```diff
@@ -368,3 +370,15 @@ def get_note(self, note_type):
         notes = [note.get('content') for note in self.notes
                  if note.get('type') == note_type]
         return next(iter(notes), None)
+
+    @property
+    def is_new_acquisition(self):
```

Comment:
I guess the item will be always considered `new_acquisition` until someone deletes the `acquisition_date` from the item record. as long as that is what is requested by the PO, no problem

Reply:
I'm pretty sure that it's the behavior that PO expected. You can filter the the date interval using the `new_acquisition` filter when you search a document. By example for 'last 6 months acquisition' : `https://localhost:5000/global/search/documents?q=&page=1&size=10&new_acquisition=now-6M%2Fd` (ES is wonderfull !! )